database/src/main/scala/cromwell/database/slick/BackendKVStoreSlickDatabase.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.database.slick
+
+import cromwell.database.sql.BackendKVStoreSqlDatabase
+import cromwell.database.sql.tables.BackendKVStore
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait BackendKVStoreSlickDatabase extends BackendKVStoreSqlDatabase {
+  this: SlickDatabase =>
+
+  import dataAccess.driver.api._
+
+
+  override def addBackendJobKeyValuePair(entries: Iterable[BackendKVStore])(implicit ec: ExecutionContext): Future[Unit] = {
+    val action = dataAccess.backendKVStoreAutoInc ++= entries
+    runTransaction(action) map { _ => () }
```

Comment:
TOL / TIL Scalaz has a `void` method on `Functor` to do the equivalent of `map { _ => () }`.  Better / worse / indifferent?

Current

```scala
runTransaction(action) map { _ => () }
```

Scalaz
```
runTransaction(action).void
```